test/list-cases/html-elements.html

```diff
@@ -0,0 +1,6 @@
+<ol>
+  <li>
+    <ins>Ins tags <var>are</var> allowed</ins>.</li>
```

Comment:
Looks like beautify is broken here; no linebreak before, ideally. If this isn't an easy fix then we can leave it.